src/js/view/controls/controlbar.js

```diff
@@ -113,10 +114,10 @@ define([
             }
             // Do not show the volume toggle in the mobile SDKs or <iOS10
             if (!_model.get('sdkplatform') && !(OS.iOS && OS.version.major < 10)) {
-                muteButton = button('jw-icon-volume', () => { _api.setMute(); }, vol);
+                muteButton = button('jw-icon-volume', () => _api.setMute(), vol);
             }
 
-            const nextButton = button('jw-icon-next', () => { _api.next(); }, next);
+            const nextButton = button('jw-icon-next', () => _api.next(), next);
```

Comment:
The brackets were intentional. Without them, the result of the statement is returned, we don't want to `return _api.setMute()` from the callback, we just want to call `_api.setMute()`.